azurerm/internal/services/network/virtual_network_gateway_resource.go

```diff
@@ -311,17 +314,57 @@ func resourceVirtualNetworkGateway() *schema.Resource {
 							Optional: true,
 						},
 
+						// TODO 3.0 - Remove this property
 						"peering_address": {
-							Type:     schema.TypeString,
-							Optional: true,
-							Computed: true,
-							ForceNew: true,
+							Type:       schema.TypeString,
+							Optional:   true,
+							Computed:   true,
+							Deprecated: "Deprecated in favor of `peering_addresses`",
```

Comment:
Could you include a little more detail here on how to move over to the new property? does this correspond to the `apipa_addresses` despite it taking multiple cross multiple peering address blocks?

Reply:
As is mentioned in the PR [description](https://github.com/terraform-providers/terraform-provider-azurerm/pull/10381#issue-564000073), this is now replaced by the `bgp_settings.0.peering_addresses.0.default_addresses.0`.

I'll extend the deprecation message to include this.